src/current/v23.1/cloud-storage-authentication.md

```diff
@@ -651,23 +651,82 @@ You can authenticate to Azure with explicit credentials in the following ways:
 Implicit authentication to Azure is only available for CockroachDB {{ site.data.products.core }} clusters.
 {{site.data.alerts.end}}
 
-{% include_cached new-in.html version="v23.1" %} If the `AUTH` parameter is set to `implicit`, credentials will be loaded from the environment (i.e., the machines running the backup) or with a managed identity. You need to set the following environment variables for `implicit` authentication:
+{% include_cached new-in.html version="v23.1.9" %} When the `AUTH` parameter is set to `implicit`, CockroachDB will load credentials from one of the following:
 
-- `AZURE_CLIENT_ID`: Application (client) ID for your [App Registration](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app#register-an-application).
-- `AZURE_CLIENT_SECRET`: Client credentials secret generated for your App Registration.
-- `AZURE_TENANT_ID`: Directory (tenant) ID for your App Registration.
+- A credentials file with the path specified in the environment variable `COCKROACH_AZURE_APPLICATION_CREDENTIALS_FILE`. Refer to [Set up a credentials file](#set-up-a-credentials-file).
+- Each credential set as an environment variable.
+    - `AZURE_CLIENT_ID`: Application (client) ID for your [App Registration](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app#register-an-application).
+    - `AZURE_CLIENT_SECRET`: Client credentials secret generated for your App Registration.
+    - `AZURE_TENANT_ID`: Directory (tenant) ID for your App Registration.
+- An Azure [managed identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview).
 
-See Microsoft's [Azure Authentication](https://learn.microsoft.com/azure/developer/go/azure-sdk-authentication) documentation to set this up. 
+You can use this for authenticating an Azure Blob Storage URI or an Azure Key Vault URI for an [Azure encrypted backup]({% link {{ page.version.version }}/take-and-restore-encrypted-backups.md %}).
 
-You must include the container name and Azure account name in your URI, as follows:
+{{site.data.alerts.callout_info}}
+You must export the environment variable on each CockroachDB node.
```

Comment:
Does this note only apply to bullet 2 regarding environmental variables? If so, should this note be after line 660?

Reply:
From line 691, I realized that the note also applies to bullet 1. Does it apply to bullet 3 as well?